lib/certUtils.js

```diff
@@ -480,7 +487,7 @@ const certMap = new Map();
 class CertManager {
 	static addCert(certBuf) {
 		const cert = new Certificate(certBuf);
-		const serial = cert.getSerial();
+		const serial = cert.getIssuer();
```

Comment:
I have one more question about `CertManager`. It seems like the internal map in `CertManager` uses the full device issuer name info (ex. "Yubico U2F EE Serial 1432534688") as keys. However, the methods of this class seem to indicate that it actually uses just the serial number (ex. the method `getCertBySerial`). I thought this could be a bit confusing, but I wasn't sure if changing it could cause more breaking changes. Do you think this should be changed?

Reply:
Oops, I just realized something. Actually it uses the full subject device common name and not the full issuer device common name. Good thing I caught this!